display.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, board_states, players):
         ]
 
         # Draw the generated board, aka first board state
-        self.initialize_board(board_states, players)
+        self.initialize_board(board_states, players, has_won, winner)
```

Comment:
What is the purpose of `has_won`? Can't you determine if they have won if the winner is set?